lib/apisExtension.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var Api = require('./api');
+
+class ApisExtension extends Api {
+  constructor(options) {
+    options.path = 'apis/extensions';
```

Comment:
Instead of mutating `options`, can we use `Object.assign` to make a copy?  e.g.,
```
options = Object.assign({}, options, {
  version: options.version || 'v1beta1',
  genericTypes: genericTypes
});
```